chef/cookbooks/provisioner/templates/default/zzz-prompt.sh.erb

```diff
@@ -7,7 +7,12 @@ fi
 # Only set the crazy prompt on terminals that support colors
 # it looks horrible on monochromatic terminals
 
-if [ $(tput colors) -ge 8 ]; then
+# Make sure that COLORS has a value even when tput does not recognize the
+# terminal type.
+COLORS=$(tput colors 2> /dev/null)
```

Comment:
Convention dictates that shell variables are only upper case iff they are either exported environment variables, or special variables set by the shell (e.g. `BASH_*`, `PWD` etc.) or used by the shell (e.g. `HOME`, `PATH`, `HISTFILE` etc.)

I don't think this falls into one of those categories, so I would suggest that it be made lowercase (i.e. `colors`).